README.md

```diff
@@ -131,6 +131,7 @@ jobs:
           - unit
     runs-on: ubuntu-22.04
     env:
+      GOBIN: /home/runner/go/bin
```

Comment:
Maybe se to use GitHub predefined variable for runner path?

Reply:
Agreed. Updated.